app/controllers/comment_controller.php

```diff
@@ -0,0 +1,55 @@
+<?php
+class CommentController extends AppController
+{
+	CONST MAX_ITEM_PER_PAGE = 10;
+
+	public function view()
+	{
+		session_start();
```

Comment:
I already moved po the session_start() in bootstrap so I'll have to call it once nalang po.
Link: https://github.com/renzosunico/BoardExercises/commit/d26fa2aa431bad368f1483ba4e3d0916fd8e9676